react/week5/githubusersprofile/src/Components/Profile.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import "./Profile.css";
+
+export function Profile({ user }) {
```

Comment:
```suggestion
export function Profile() {
```
Can't see why you need user